test/src/test/java/hudson/model/JobTest.java

```diff
@@ -415,58 +415,51 @@ private String relativePath(File base, FilePath path) throws IOException, Interr
     @Issue("JENKINS-30502")
     @Test
     public void testRenameTrimsLeadingSpace() throws Exception {
-        tryRename("myJob1", " foo", "foo", false);
+        tryRename("myJob1", " foo", "foo");
     }
 
     @Issue("JENKINS-30502")
     @Test
     public void testRenameTrimsTrailingSpace() throws Exception {
-        tryRename("myJob2", "foo ", "foo", false);
+        tryRename("myJob2", "foo ", "foo");
     }
 
     @Issue("JENKINS-30502")
     @Test
     public void testAllowTrimmingByUser() throws Exception {
         assumeFalse("Unix-only test.", Functions.isWindows());
-        tryRename("myJob3 ", "myJob3", "myJob3", false);
+        tryRename("myJob3 ", "myJob3", "myJob3");
     }
 
     @Issue("JENKINS-30502")
     @Test
     public void testRenameWithLeadingSpaceTrimsLeadingSpace() throws Exception {
         assumeFalse("Unix-only test.", Functions.isWindows());
-        tryRename(" myJob4", " foo", "foo", false);
+        tryRename(" myJob4", " foo", "foo");
     }
 
     @Issue("JENKINS-30502")
     @Test
     public void testRenameWithLeadingSpaceTrimsTrailingSpace()
             throws Exception {
         assumeFalse("Unix-only test.", Functions.isWindows());
-        tryRename(" myJob5", "foo ", "foo", false);
+        tryRename(" myJob5", "foo ", "foo");
     }
 
     @Issue("JENKINS-30502")
     @Test
     public void testRenameWithTrailingSpaceTrimsTrailingSpace()
             throws Exception {
         assumeFalse("Unix-only test.", Functions.isWindows());
-        tryRename("myJob6 ", "foo ", "foo", false);
+        tryRename("myJob6 ", "foo ", "foo");
     }
 
     @Issue("JENKINS-30502")
     @Test
     public void testRenameWithTrailingSpaceTrimsLeadingSpace()
             throws Exception {
         assumeFalse("Unix-only test.", Functions.isWindows());
-        tryRename("myJob7 ", " foo", "foo", false);
-    }
-
-    @Issue("JENKINS-30502")
-    @Test
-    public void testDoNotAutoTrimExistingUntrimmedNames() throws Exception {
-        assumeFalse("Unix-only test.", Functions.isWindows());
-        tryRename("myJob8 ", "myJob8 ", null, true);
```

Comment:
This test only made sense when renames were handled by `/configure`. That code has been deleted so unintended renames when configuring other settings should not be a problem.